src/convert.cpp

```diff
@@ -101,13 +125,22 @@ ddwaf_object* to_ddwaf_string(ddwaf_object *object, Napi::Value val, bool lim) {
   }
   return ddwaf_object_stringl(object, str.c_str(), len);
 }
-
+ddwaf_object* to_ddwaf_object(
+        ddwaf_object *object,
+        Napi::Env env,
+        Napi::Value val,
+        int depth,
+        bool lim
+) {
+  return to_ddwaf_object(object, env, val, depth, lim, false);
+}
```

Comment:
can't we have a default value instead ? there: https://github.com/DataDog/dd-native-appsec-js/blob/main/src/convert.h#L19

Reply:
ooh, I thought that it wasn't possible in c++

Reply:
Aaah much cleaner like that perfect :+1: